runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -151,12 +154,18 @@ def ready(self, value: bool):
         self._rt.ready = value
 
     def decode(
-        self, request_input: RequestInput, default_codec: Optional[InputCodec] = None
+        self,
+        request_input: RequestInput,
+        default_codec: Optional[Union[Type[InputCodec], InputCodec]] = None,
```

Comment:
nit: should we have an alias `Union[Type[InputCodec], InputCodec]` as it is being used in many places?

Reply:
Good point, let me add an alias for that.